build.csx

```diff
@@ -98,8 +100,8 @@ targets.Add(
             var form = new MultipartFormDataContent();
             form.Add(new StringContent(coverityToken), @"""token""");
             form.Add(new StringContent(coverityEmail), @"""email""");
-            form.Add(new StringContent(version), @"""version""");
-            form.Add(new StringContent($"Build {version} ({repoCommitId})"), @"""description""");
+            form.Add(new StringContent(repoCommitId), @"""version""");
+            form.Add(new StringContent($"Build ({repoCommitId})"), @"""description""");

```

Comment:
Rather than jump through hoops figuring out the version, I figured we could just use the commit ID.
Happy to look for an alternative if you don't like it.